modules/commandModule/commandWorker_flight.py

```diff
@@ -15,4 +19,34 @@ def flight_command_worker(pipelineIn, pipelineOut):
 
 	#TODO: Write logic to get POGI
 
-	
\ No newline at end of file
+
+def pogi_command_worker(pause, exitRequest, pipelineOut):
+    """
+    Producer worker function to return data currently in the POGI file
+
+    Parameters
+    ----------
+    pause : multiprocessing.Lock
+        a lock shared between worker processes
+    exitRequest : multiprocessing.Queue
+        a queue that determines when the worker process stops
+    pipelineOut : multiprocessing.Queue
+        output pipeline for pogiData
+    """
+
+    print("Reading POGI data")
+
+    while True:
+        pause.acquire()
+        pause.release()
+
+        # get the pogi data
+        pogiData = read_pogi()
```

Comment:
I'm going to slightly expand the scope of this ticket, I apologize for that, but I think there's a better way to do this. 

Essentially, we're going to need to resuse this logic in every worker function (one for flight, one for taxi, one for search). Therefore, we should pull it out into its own function. Instead of having this worker function, write something called `pogi_subworker(pipelineOut, POGI_DIR)`. This should be a function that encompasses line 43 - 49, just the logic to read the latest pogi data, compare it against the last one, and put it in the pipeline. 

Then, we should be able to call this in the other worker functions, so each time those worker functions run, they first perform whatever activities they're supposed to do, then they update their pipelineOut with the latest pogi data.